locale/ar/get-involved/node-events.md

```diff
@@ -0,0 +1,20 @@
+﻿---
+title: مناسبات الـ Node.js
```

Comment:
مناسبات > فعاليات ؟

Reply:
> 
> 
> Hi @celyes：
> Like 'node-speakers.md', you DON'T need to translate 'node-events.md' because this file MUST BE submitted in English, on the other hand, this file will be frequently changed.
> 
> For related things, please refer to this: #2022
Thank you so much, but since it's OK for now as @fhemberger said in #2022, we will keep it updated in Arabic every time the English version gets an update.